### PR TITLE
LFT_PCR Blurb

### PR DESCRIPTION
**Pull request by RLordan**
Hey @rando2, here is my two cents worth. It might actually be better mixed in with what @aimundo just wrote, but I will leave it to you!

### CODE REVIEW

**Comment by agitter** [comment]
Once we merge in #1106 and sync it here, that should help considerably with the build times.

File: content/10.diagnostics.md [hunk 1]
```diff
@@ -511,6 +511,20 @@ Immunoassays, including serological tests, have much lower limits of detection t
 LOD between tests: 10.1056/NEJMcp2117115
 !--Add text citing papers from Ronan & Nils--> 
 
+Changes in public attitudes and the lifting of COVID-19 restrictions due to the multifactorial desire to stimulate economic activities has required a shift of testing paradigms in 2022, despite warnings from public health officials of a hard exit from public health restrictions [@doi:10.1136/bmj.o383; @doi:10.1186/s12889-022-12845-2].
+An important strategy for testing moving forward is to determine when someone becomes infectious or is no longer infectious following a positive test for COVID-19. 
+Generally, patient specimens tend to not contain culturable virus past day 5 of symptom onset [@doi:10.1093/cid/ciaa1764; @doi:10.1016/S2666-5247(20)30172-5].
```
Comment (agitter) [inline content/10.diagnostics.md:516]:
```suggestion
Generally, patient specimens tend to not contain culturable virus past day 5 of symptom onset [@doi:10.1093/cid/ciaa1764; @{doi:10.1016/S2666-5247(20)30172-5}]. 
```
This should help parse the DOI with a parenthesis in it.

File: content/10.diagnostics.md [hunk 2]
```diff
@@ -511,6 +511,20 @@ Immunoassays, including serological tests, have much lower limits of detection t
 LOD between tests: 10.1056/NEJMcp2117115
 !--Add text citing papers from Ronan & Nils--> 
 
+Changes in public attitudes and the lifting of COVID-19 restrictions due to the multifactorial desire to stimulate economic activities has required a shift of testing paradigms in 2022, despite warnings from public health officials of a hard exit from public health restrictions [@doi:10.1136/bmj.o383; @doi:10.1186/s12889-022-12845-2].
+An important strategy for testing moving forward is to determine when someone becomes infectious or is no longer infectious following a positive test for COVID-19. 
+Generally, patient specimens tend to not contain culturable virus past day 5 of symptom onset [@doi:10.1093/cid/ciaa1764; @{doi:10.1016/S2666-5247(20)30172-5}]. 
+However, PCR-based methods due to their sensitivity to post-infectious viral RNA in specimens may mislead individuals to believe that they are still infectious several days after symptom onset [@:10.1056/NEJMp2025631].
```
Comment (rando2) [inline content/10.diagnostics.md:517]:
```suggestion
However, PCR-based methods due to their sensitivity to post-infectious viral RNA in specimens may mislead individuals to believe that they are still infectious several days after symptom onset [@doi:10.1056/NEJMp2025631]. 
```

File: content/10.diagnostics.md [hunk 3]
```diff
@@ -511,6 +511,20 @@ Immunoassays, including serological tests, have much lower limits of detection t
...
+However, PCR-based methods due to their sensitivity to post-infectious viral RNA in specimens may mislead individuals to believe that they are still infectious several days after symptom onset [@:10.1056/NEJMp2025631]. 
+Furthermore, detection of viral RNA can occur days and weeks after an active infection due to the sensitivity of PCR-based methods [@doi:10.1038/s41420-020-00375-y; @doi:10.1038/s41467-020-20568-4; @doi:10.1001/jamanetworkopen.2021.42796]. 
+
+In contrast, LFTs were thought to have poor sensitivity and their value for identifying infections and managing the pandemic was questioned [@doi:10.1136/bmj.m4941; @doi:10.1136/bmj.n81].
+However, LFTs can detect SARS-CoV-2 proteins when there is a high viral load, which appears to correlate with a person’s infectiousness [@doi:10.1128/jcm.02479-21; @doi:10.1056/NEJMcp2117115]. 
+Therefore, LFTs are an important diagnostic tool to determine infectiousness with fast turnaround times, ease of use, and accessibility by the general public  [@doi:10.1016/S0140-6736(21)00425-6; @doi:10.1056/NEJMp2025631].
```
Comment (rando2) [inline content/10.diagnostics.md:522]:
```suggestion
Therefore, LFTs are an important diagnostic tool to determine infectiousness with fast turnaround times, ease of use, and accessibility by the general public  [@doi:10/gnmbjd; @doi:10.1056/NEJMp2025631].
```

File: content/10.diagnostics.md [hunk 4]
```diff
@@ -511,6 +511,20 @@ Immunoassays, including serological tests, have much lower limits of detection t
...
+One study has suggested that the test sensitivity of LFTs appears to be less important than accessibility to LFTs, frequent testing, and fast reporting times for reducing the impact of viral spread [@doi:10.1126/sciadv.abd5393].
+While PCR-based methods are important for COVID-19 surveillance, its use is labor intensive, time consuming, and laboratories are often slow to report results rendering the method limited in its surveillance capacity [@doi:10.1056/NEJMp2025631] as demonstrated by the estimated 10-fold underreporting of cases in the United States in 2020 due to shortages in testing and slow rollout of testing and slow reporting of results [@url:https://www.washingtonpost.com/health/2020/06/25/coronavirus-cases-10-times-larger/l].
```
Comment (rando2) [inline content/10.diagnostics.md:524]:
```suggestion
While PCR-based methods are important for COVID-19 surveillance, their use is labor intensive and time consuming, and laboratories are often slow to report results, rendering such methods limited in their surveillance capacity [@doi:10.1056/NEJMp2025631].
These limitations are demonstrated by the estimated 10-fold under-reporting of cases in the United States in 2020 due to shortages in testing and slow rollout of testing and slow reporting of results [@url:https://www.washingtonpost.com/health/2020/06/25/coronavirus-cases-10-times-larger].  
```

File: content/10.diagnostics.md [hunk 5]
```diff
@@ -511,6 +511,20 @@ Immunoassays, including serological tests, have much lower limits of detection t
...
+Therefore, LFTs are an important diagnostic tool to determine infectiousness with fast turnaround times, ease of use, and accessibility by the general public  [@doi:10.1016/S0140-6736(21)00425-6; @doi:10.1056/NEJMp2025631].
+One study has suggested that the test sensitivity of LFTs appears to be less important than accessibility to LFTs, frequent testing, and fast reporting times for reducing the impact of viral spread [@doi:10.1126/sciadv.abd5393].
+While PCR-based methods are important for COVID-19 surveillance, its use is labor intensive, time consuming, and laboratories are often slow to report results rendering the method limited in its surveillance capacity [@doi:10.1056/NEJMp2025631] as demonstrated by the estimated 10-fold underreporting of cases in the United States in 2020 due to shortages in testing and slow rollout of testing and slow reporting of results [@url:https://www.washingtonpost.com/health/2020/06/25/coronavirus-cases-10-times-larger/l].  
+However, one strategy is to corroborate a postive LFT result using a PCR-based method.
```
Comment (rando2) [inline content/10.diagnostics.md:525]:
```suggestion
However, one strategy that may balance the strengths and weaknesses of both types of tests is to corroborate a positive LFT result using a PCR-based method.
```

File: content/10.diagnostics.md [hunk 6]
```diff
@@ -511,6 +511,20 @@ Immunoassays, including serological tests, have much lower limits of detection t
...
+Indeed, sufficient surveillance and reduction of COVID-19 spread was demonstrated in Liverpool where there was an estimated 21% reduction of cases in December 2021 using this LFT-PCR strategy [@url:https://www.gov.uk/government/publications/liverpool-coronavirus-covid-19-community-testing-pilot-full-evaluation-report-summary/liverpool-coronavirus-covid-19-community-testing-pilot-full-evaluation-report-summary; @doi:10.1016/S0140-6736(21)00425-6].
```
Comment (rando2) [inline content/10.diagnostics.md:526]:
```suggestion
Indeed, in December 2021 sufficient surveillance and reduction of COVID-19 spread using this joint LFT-PCR strategy was demonstrated in Liverpool, U.K., where there was an estimated 21% reduction of cases [@url:https://www.gov.uk/government/publications/liverpool-coronavirus-covid-19-community-testing-pilot-full-evaluation-report-summary/liverpool-coronavirus-covid-19-community-testing-pilot-full-evaluation-report-summary; @doi:10/gnmbjd]. 
```

**Review by rando2 — APPROVED** [review]
This looks great to me @RLordan!

File: content/10.diagnostics.md (hunk 3 above)
Comment (RLordan) [reply content/10.diagnostics.md:520]: Typical me messing up references! :D

**Comment by RLordan** [comment]
> This looks great to me @RLordan!

Thanks @rando2 I think I approved all commits.

**Pull request merged**